tests/test_upload_photo.py

```
@@ -0,0 +1,126 @@
+    b'\x89PNG\r\n\x1a\n'
+    b'\x00\x00\x00\rIHDR\x00\x00\x00\x01\x00\x00\x00\x01\x08\x02'
+    b'\x00\x00\x00\x90wS\xde'
+    b'\x00\x00\x00\x0cIDATx\x9cc\xf8\x0f\x00\x00\x01\x01\x00\x05\x18\xd8N'
+    b'\x00\x00\x00\x00IEND\xaeB`\x82'
+)
+
+
+@pytest.fixture
+async def auth_token(service_client):
+    email = make_unique_email()
+    password = 'TestPassword123'
```

Ruff (0.15.15)

[error] Possible hardcoded password assigned to: "password"

(S105)